c7n/utils.py

```diff
@@ -375,3 +376,8 @@ def reformat_schema(model):
             ret[key]['required'] = True
 
     return ret
+
+
+def get_available_regions(resource):
```

Comment:
@kapilt  - I've cleaned this up, and went one step further and made a common session in utils that is shared by cli.py and policy.py.  (This is something we talked about fixing from my account-id PR)

(If you don't like this approach, I could make the session in cli.py and pass it via `options` to policy.py.)